src/Components/ModalPlaylist/ModalPlaylist.jsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { useState } from "react";
+import { useVideo } from "../../Context/VideoContext";
+import { addPlaylist } from "../../Services/Playlist/addPlaylist";
+import { addPlaylistVideo } from "../../Services/Playlist/addPlaylistVideo";
+import { removePlaylistVideo } from "../../Services/Playlist/removePlaylistVideo";
+import "./modalplaylist.css";
+
+function ModalPlaylist({ item, setShowModal }) {
+  const { stateVideo, dispatchVideo } = useVideo();
+  const { playlist } = stateVideo;
+  const [playlistTitle, setPlaylistTitle] = useState("");
+
+  const playlistHandler = () => {
+    addPlaylist(playlistTitle, dispatchVideo);
+    setPlaylistTitle("");
+  };
+
+  const addPlaylistVideoHandler = (id) => {
+    addPlaylistVideo(id, item, dispatchVideo);
+  };
+
+  const removePlaylistVideoHandler = (id) => {
+    removePlaylistVideo(id, item._id, dispatchVideo);
+  };
+
+  // const [alreadyInPlaylist, setAlreadyInPlaylist] = useState(false);
```

Comment:
remove commented code if you're not going to use it.

Reply:
Thanks